### PR TITLE
add variant_class and lib--> exp strat

### PR DESCRIPTION
**Pull request by rosibaj**
- changed `library_strategy` to `experimental_strategy`
- added `variant_class` to file_centric only.  I dont see a use case for being in analysis_centric right now

### CODE REVIEW

File: mappings/maestro_analysis_centric.json [hunk 1]
```diff
@@ -82,7 +82,7 @@
           "platform": {
             "type": "keyword"
           },
-          "library_strategy": {
```
Comment (blabadi) [inline mappings/maestro_analysis_centric.json:85]: do the same for the platform file centric index ?

Comment (blabadi) [reply mappings/maestro_analysis_centric.json:85]: this affects rdpc, we should include Alexis/Alex too or post this in rdpc dev

Comment (blabadi) [reply mappings/maestro_analysis_centric.json:85]: you already did post this, thanks

Comment (rosibaj) [reply mappings/maestro_analysis_centric.json:85]: resolving this comment as 
- it is changed below 
- i will make sure on slack both alex and alexis see the change, in addition to the current post.

File: mappings/maestro_analysis_centric.json [hunk 2]
```diff
@@ -82,15 +82,15 @@
...
+          "experimental_strategy": {
             "type": "keyword"
           }
         }
       },
       "study_id": {
         "type": "keyword",
         "copy_to": ["analysis_autocomplete"]
-      },
+      }
```
Comment (blabadi) [inline mappings/maestro_analysis_centric.json:93]: comma needed here

**Comment by junjun-zhang** [comment]
`data_category` and `analysis_tools` under `file.info` are meant to be the same level as `file.dateType` in SONG, but since we cannot change SONG's base schema, we added them to under `info`.

So in terms of ES mapping, `data_category` and `analysis_tools` should be added to wherever `data_type` field goes.

**Comment by rosibaj** [comment]
@junjun-zhang as mentioned here: https://github.com/icgc-argo/argo-metadata-schemas/issues/32#issuecomment-640863705  this PR was only to handle the `library_strategy` and the `variant_class` fields. 

We need to do a bit of additional work with maestro to allow for indexing of Info fields from the base schema.

**Comment by junjun-zhang** [comment]
I am just making a comment on the other two fields, whenever they get added it's fine.

**Comment by rosibaj** [comment]
@junjun-zhang ok great - will just move that over to the main ticket then to make sure it does not get lost!

**Comment by junjun-zhang** [comment]
for this PR, should we have `variant_class` under `files` of analysis_centric?

**Comment by rosibaj** [comment]
@junjun-zhang the analysis_centric was primary built to be used by the rdpc to integrate with clinical information - i had trouble identifying a use case for `variant_class` from that perspective... is that something that we would check for workflows? (@lepsalex ?)

**Comment by junjun-zhang** [comment]
In general, when there are `a-centric` and `b-centric` indexes, they are representing same informaion but with different structures.

```
a1:
  - b1
  - b2
```
vs
```
b1:
  - a1

b2:
  - a1
```

In the case of analysis_centric, if you'd like search for `analyses` that produce data file with certain `variant_class`, you will then need to have `variant_class` nested in `analysis_centric` index

**Comment by lepsalex** [comment]
it may not be something that we check in workflows however the data would still be in the source. We just wouldn't need it in the `analysis-centric` index since I don't believe that we would be executing any queries against it. If that changes in the future we can add it.

**Pull request merged**